hatchways/team-gingerbread
# FE Profile Payment

### What this PR does (required):
- addresses #49
- Created ProfilePaymentTab in EditProfile to allow users to add credit cards to their profile

### Screenshots / Videos (front-end only):
![Screenshot](https://user-images.githubusercontent.com/78042694/145655275-37659036-ccf9-4223-8c34-2b3a778159da.png)
![Screenshot (1)](https://user-images.githubusercontent.com/78042694/145655282-d233db1c-2f47-4377-81d5-973f6b058fd7.png)
![Screenshot (2)](https://user-images.githubusercontent.com/78042694/145655287-790dbcbf-ec52-4ee6-8c06-de57cb38c01d.png)
![Screenshot (3)](https://user-images.githubusercontent.com/78042694/145655293-e4491834-8be0-4445-b315-58eb6c48dc22.png)


### Any information needed to test this feature (required):
- after logging in, navigate to EditProfile, then Payment tab
- click "Add" button
- fill out form, then click "Save" button
- your new card should be displayed in the "Saved Payment" section

### Any issues with the current functionality (optional):
- need to add in functionality for choosing default card
- currently, card type (e.g. Mastercard, VISA) is random. need to figure out how to determine type based on card number
